.gitignore

```diff
@@ -109,3 +109,6 @@ ENV/
 
 # scrapy deltafetch middleware directory
 data/deltafetch
+
+# vscode
+.vscode
```

Comment:
can you remove this? is not related with the PR

Reply:
Yes! I have changed!